apps/svelte.dev/src/routes/+layout.server.ts

```diff
@@ -51,21 +51,21 @@ const nav_links: NavigationLink[] = [
 	}
 ];
 
-// const banner: BannerData = {
-// 	id: 'adventofsvelte2024',
-// 	start: new Date('1 December, 2024 00:00:00 UTC'),
-// 	end: new Date('25 December, 2024 23:59:59 UTC'),
-// 	arrow: true,
-// 	content: {
-// 		lg: 'Twenty-four days, twenty-four features: Advent of Svelte 2024',
-// 		sm: 'Advent of Svelte 2024'
-// 	},
-// 	href: '/blog/advent-of-svelte'
-// };
+const banner: BannerData = {
+	id: 'barcelona',
```

Comment:
Maybe add a date just in case we go to Barcelona again next year 😄 
```suggestion
	id: 'barcelona-2025',
```